### DISCUSSION TITLE
Fever API with Unread

### DISCUSSION THREAD

**Discussion by AlexGeek77** [Athou/commafeed#1188] [General]
I'm having trouble connecting my Commafeed instance to the Unread feed reader app on iOS. I've set up an API key in the settings and then used the API link Unread which gives me this error message:

![image](https://github.com/Athou/commafeed/assets/1232415/22051517-1070-4c3d-b7c1-53403350b9a3)

[[Unread: An RSS Reader](https://apps.apple.com/us/app/unread-an-rss-reader/id1363637349)](https://apps.apple.com/us/app/unread-an-rss-reader/id1363637349)

**Comment by Athou** [comment]
Unread is using GET to access the Fever API, even though the fever API documentation [only mentions POST](https://web.archive.org/web/20221221112459/https://feedafever.com/api).

I added a workaround for this in https://github.com/Athou/commafeed/commit/d7b2c5a6e3e7c71ffc7276e3f9c4c627f357fbe9 can you confirm it works? You can get the jar with the fix [here](https://github.com/Athou/commafeed/actions/runs/7520747693) or if you're using docker it is in the `athou/commafeed:master` tag

**Reply by AlexGeek77** [reply]
Sorry for the delay but I'm finally able to attempt the changes. I was able to get to the login screen for the fever API but when I typed my email and password I got this error. According to [this](https://github.com/Athou/commafeed/issues/1126#issuecomment-1693865401) post indicates the login is supposed to be the username and password not email and password (though I tried both). Here is my [Pastebin](https://github.com/Athou/commafeed/commit/d7b2c5a6e3e7c71ffc7276e3f9c4c627f357fbe9) of my DEBUG level logs when I click `SIGN IN` in the Unread app.

![image](https://github.com/Athou/commafeed/assets/1232415/9a5c0181-a81c-4e88-9b32-1dc410ceb889)

**Reply by Athou** [reply]
You need to login with your username and _API key_, not password (as indicated in the post you linked 😄).

**Reply by AlexGeek77** [reply]
Oh boy. Completely missed that part. Thanks. 

Do you plan to roll this change into future releases?

**Reply by Athou** [reply]
> Do you plan to roll this change into future releases?

Yes, it will be part of v4.2, should be released this week.

**Reply by Athou** [reply]
I just release 4.2.0 :)

**Reply by AlexGeek77** [reply]
Well that was fast. Everything works with this change. Thanks for your help.
